sdk/voiceservices/Azure.ResourceManager.VoiceServices/src/autorest.md

```diff
@@ -0,0 +1,49 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+azure-arm: true
+csharp: true
+library-name: VoiceServices
+namespace: Azure.ResourceManager.VoiceServices
+require: https://github.com/Azure/azure-rest-api-specs/blob/7b47689d4efc098f25f46781f05f22179c153314/specification/voiceservices/resource-manager/readme.md
+output-folder: $(this-folder)/Generated
+clear-output-folder: true
+skip-csproj: true
+modelerfour:
+  flatten-payloads: false
+tag: package-2023-01-31
```

Comment:
since the default tag in the above readme is exactly this tag, could we just remove this line?
This helps our SDK automation pipeline in the swagger review process to keep the SDK automatically up-to-date and we could preview the SDK changes on a swagger PR